pkg/features/feature.go

```diff
@@ -30,6 +30,15 @@ type (
 	Level   = types.Level
 )
 
+const (
+	// LevelSetup when doing the setup phase
+	LevelSetup Level = types.LevelSetup
```

Comment:
```suggestion
	LevelSetup = types.LevelSetup
```
This should be good enough ?